Rocket.Chat/Controllers/Chat/MessagesSizingViewModel.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MessagesSizingViewModel.swift
+//  Rocket.Chat
+//
+//  Created by Rafael Streit on 25/09/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+final class MessagesSizingViewModel {
```

Comment:
TBH, I don't know if it should be a model (Manager) used by the ViewModel to process along with the other models that are necessary to bind the MessagesView or if it should actually be inside another ViewModel. Or even be a separated ViewModel as you placed it. 🤔 💭

Reply:
✅